### PR TITLE
[Feature]: Change Navbar items

### LINKED ISSUE #134: [FEATURE] Change the navbar items
### Detailed description

OVERVIEW | DAO | Community | Help | Launch App

**Overview**
| For Creators
| For Patrons
| For Developers
| [Roadmap](https://trello.com/b/oFu5uWJF/roadmap)

**DAO**
| [Voting Portal](https://govern.devprotocol.xyz/)

**Community**
| Forum
| Builders Program

**Help**
| FAQ
| Tutorials
| Documentation
| Code of Conduct

**Launch App** = Button for [Stakes.social](http://stakes.social/)

### Context

Dev Protocol team decided to change the navbar items with the ideas above ☝️ 

### Possible implementation

You can get the navbar codes & ideas here 👉 https://tailwindui.com/components/marketing/page-examples/pricing-pages

Please center the OVERVIEW | DAO | Community | Help and the launch app should separate to the right 
image

### Additional information

Please comment here if you want to work with this issue before making PR.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by AbhinavMV**
#### Description of the change

Changed Navbar Design
Removed Image on main page (Image with join button)

Closes #134 
#### Screenshots
image
image
image
image

#### Checklist

- [x] Added description of the change
- [x] I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md)
- [x] Search previous suggestions before making a new PR, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/141"></a>

### CODE REVIEW

**Review by vinzvinci — COMMENTED** [review]
Can launch app seen in nav item of mobile and tablet view?

**Comment by AbhinavMV** [comment]
Is this ok ? 
image

**Comment by vinzvinci** [comment]
> Is this ok ? image

Hmm how about make it text only not button, just don't add dropdown arrow.

**Comment by AbhinavMV** [comment]
> > Is this ok ? image
> 
> Hmm how about make it text only not button, just don't add dropdown arrow.

Done
image

**Comment by vinzvinci** [comment]
Looks good @AbhinavMV

**Review by gptkrsh — APPROVED** [review]

**Review by vinzvinci — APPROVED** [review]

File: public/assets/dropdown-arrow.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M19 9l-7 7-7-7" />
+</svg>
```
Comment (Panquesito7) [inline public/assets/dropdown-arrow.svg:3]: Missing an endline.
```suggestion
</svg>

```

**Pull request merged**